src/Symfony/Component/HttpFoundation/EmptyResponse.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation;
+
+class EmptyResponse extends Response
+{
+    /**
+     * @param array $headers
+     */
+    public function __construct(array $headers = [])
+    {
+        parent::__construct(null, Response::HTTP_NO_CONTENT, $headers);
+    }
+
+    /**
+     * Factory method for chainability.
+     *
+     * Example:
+     *
+     *     return Response::create()
```

Comment:
```suggestion
     *     return EmptyResponse::create() 
```

??

and btw is this comment helpful?

Reply:
:+1: for `EmptyResponse`, for your other question I just did the same as the other `Response` classes